encoders/.gitignore

```diff
@@ -0,0 +1,20 @@
+*.class
+
+# sbt specific
+.cache/
+.history
+.lib/
+dist/*
+lib_managed/
+src_managed/
+project/boot/
+project/plugins/project/
+
+# Scala-IDE specific
+.scala_dependencies
+.worksheet
+.idea
+
+# Emacs files
+.#*
+*~
```

Comment:
Does this need a separate .gitignore? The top-level one should work.

Reply:
We don't need it, as encoders part of the same top-level snappydata repository, not a separate one.